src/content/refi/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Regenerative Finance (ReFi)
+description: An overview of ReFi and current use cases.
+lang: en
+template: use-cases
+emoji: ":recycle:"
+sidebarDepth: 2
+image: ../../assets/future_transparent.png
+alt: ""
+summaryPoint1: An alternative economic and funding system focused on sustainability.
+summaryPoint2: A tool used to drastically scale up current, slow voluntary carbon markets.
+summaryPoint3: Attempts to mitigate and/or reduce the impact of global and regional crises.
+---
+
+## The Roots of ReFi: Regenerative Capitalism and the Cypherpunks {#roots-of-refi}

```

Comment:
Just noting, we typically use sentence casing throughout the site for headers (capitalizing only first word and proper nouns)

Reply:
```suggestion
## The roots of ReFi: regenerative capitalism and the cypherpunks {#roots-of-refi}
```